Styling/README.md

```diff
@@ -141,6 +142,27 @@ Since `expression` is optional and conditions are evaluated in order, the above
 }
 ```
 
+Non-visual properties of a feature can be defined using the `meta` property. 
+
+For example, to set a `description` meta property to a string containing the feature name:
+```json
+{
+    "meta" : {
+        "description" : "'Hello, ${featureName}.'"
+    }
+}
+```
+
+A meta property expression can evaluate to any type. For example:
+```json
+{
+    "meta" : {
```

Comment:
No need to change anything yet, but I almost feel like these are basically declaring variables and it would be cool to be able to use them in expressions like we do in `conditions`.  Let's see how it plays out.